### PR TITLE
Fixed remaining flake8 errors in rest

### PR DESCRIPTION
**Pull request by hiteshhapani**

### CODE REVIEW

File: gbpservice/nfp/configurator/api/v1/controllers/controller.py [hunk 1]
```diff
@@ -103,6 +102,8 @@ def post(self, **body):
             pecan.response.status = 400
             msg = ("Failed to serve HTTP post request %s %s."
                    % (self.method_name, str(err).capitalize()))
+            extra_import = ("need to remove this import %s" % config)
```
Comment (njagadish) [inline gbpservice/nfp/configurator/api/v1/controllers/controller.py:105]: We should try to avoid this. Can you check how it is working for config agent?

Comment (njagadish) [inline gbpservice/nfp/configurator/api/v1/controllers/controller.py:105]: Did you check how this is working for config agent? Closing this PR is dependent on this.

Comment (hiteshhapani) [inline gbpservice/nfp/configurator/api/v1/controllers/controller.py:105]: config-agent module is restructured so I looked into other files. In main.py this import is there but it is using that and in other files which are making rpc call if they are not using this they are not importing it. I have checked removing this import  and maintaining this import in both case it sends same rpc call object to configurator. But need to be checked that configurator behaves same in both case or not.

**Comment by njagadish** [comment]
Merging this pull to fix the flake8 errors in mitaka branch. Need to address the pending import issue for which we put a workaround now.

**Pull request merged**